src/data/map.js

```diff
@@ -370,7 +370,7 @@ const getGyms = async (minLat, maxLat, minLon, maxLon, updated, raidsOnly, showR
         if (excludedLevels.length === 0) {
             excludeLevelSQL = '';
         } else {
-            let sqlExcludeCreate = 'AND (raid_level NOT IN (';
+            let sqlExcludeCreate = 'AND (raid_pokemon_id <= 0 OR raid_level NOT IN (';
```

Comment:
```suggestion
            let sqlExcludeCreate = 'AND (raid_pokemon_id > 0 OR raid_level NOT IN (';
```
Making the following changes seem to do your intended filtering. Otherwise it would only show eggs for me with only Unown raids enabled.

Reply:
Lol yes sorry, this one was untested change.